jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -169,31 +168,35 @@ private fun JTextArea.speedSearchHighlighter(speedSearchEnabledComponent: JCompo
     }
 }
 
-class WrappingCellRenderer(private val wrapOnSelection: Boolean, private val toggleableWrap: Boolean) : DefaultTableCellRenderer() {
+class WrappingCellRenderer(private val wrapOnSelection: Boolean, private val toggleableWrap: Boolean) : CellRendererPanel(), TableCellRenderer {
     var wrap: Boolean = false
 
-    // JBTextArea has a different font from JBLabel (the default in a table) so harvest the font off of it
-    private val jLabelFont = JBLabel().font
+    private val textArea = JBTextArea()
+
+    init {
+        textArea.font = UIUtil.getLabelFont()
+        textArea.border = JBUI.Borders.empty(2, 2)
+        textArea.wrapStyleWord = true
+
+        add(textArea)
+    }
 
     override fun getTableCellRendererComponent(table: JTable?, value: Any?, isSelected: Boolean, hasFocus: Boolean, row: Int, column: Int): Component {
-        val defaultComponent = super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column)
-        table ?: return defaultComponent
-        val component = JBTextArea()
-
-        component.border = (defaultComponent as? JLabel)?.border ?: JBUI.Borders.empty(2, 2)
-        component.wrapStyleWord = (wrapOnSelection && isSelected) || (toggleableWrap && wrap)
-        component.lineWrap = (wrapOnSelection && isSelected) || (toggleableWrap && wrap)
-        component.font = jLabelFont
-        component.text = (value as? String)?.trim()
-        component.setSelectionHighlighting(table, isSelected)
-
-        component.setSize(table.columnModel.getColumn(column).width, component.preferredSize.height)
-        if (table.getRowHeight(row) != component.preferredSize.height) {
-            table.setRowHeight(row, component.preferredSize.height)
+        if (table == null) {
+            return this
+        }
+
+        textArea.lineWrap = (wrapOnSelection && isSelected) || (toggleableWrap && wrap)
+        textArea.text = (value as? String) ?: ""
```

Comment:
Removed the trim? The reason the trim was there was for when you end up with messages,  like "abc\n\n" which happen, but maybe it's better this way?

Reply:
Eh, if we want to keep the trim, we should trim before the table model, not every paint call.

If you think we should keep it, I can go hunt that down.

Reply:
(with wrapping turned off)
<img width="1068" alt="Screen Shot 2020-08-19 at 7 53 10 AM" src="https://user-images.githubusercontent.com/11964782/90650823-2d346d80-e1f1-11ea-9006-db9b4f7986cf.png">
Can we re-add trim (and add a comment that this is why)